### PR TITLE
Building hours

### PR DESCRIPTION
**Pull request by zimmer18**
Add building hours for
- OSA
- Conferences, Camps and Events
- Parking office
- Admissions
- Piper Center
- Business Office
- Registrar
- Financial Aid
- Pastor's Office
- CMIE

and remove all extra apostrophes from all the building hour yaml files

### CODE REVIEW

**Comment by zimmer18** [comment]
Closes #544

File: data/building-hours/5-2-asc.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-name: 'Academic Support Center'
-abbreviation: 'ASC'
-category: 'Help and Support'
+name: ASC
+subtitle: Academic Support Center
```
Comment (hawkrives) [inline data/building-hours/5-2-asc.yaml:2]: Why flip this?

File: data/building-hours/9-osa.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: OSA
+subtitle: Office of Student Activities
```
Comment (hawkrives) [inline data/building-hours/9-osa.yaml:2]: This feels like it should be Name and Abbreviation instead of Name and Subtitle...

**Review by hawkrives — COMMENTED** [review]
I remarked on two places in the diff. 

It's entirely subjective, but some buildings I think about as the abbreviation, and some I think of by their name. 

Buildings with really long names should probably have the full name as the subtitle, like the CMIE, but the ASC and the OSA should, IMHO, be Name (Abbr). 

In part, that's a judgement call around how common the abbreviation is. I don't think of OSA as OSA, since I was never in an elected position (actually, I think of it as the SGA room, but that's not an option here.)

I'm rambling. Point is, I'd like to hear other people's opinions on those two rooms. 

Thanks for doing this!

File: data/building-hours/9-piper-center.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+name: Piper Center
+category: Offices
+
+schedule:
+  - title: Hours
+    hours:
+      - {days: [Mo, Tu, We, Th, Fr], from: '8:00am', to: '5:00pm'}
+      - {days: [Tu, We], from: '7:00pm', to '9:00pm'}
```
Comment (hawkrives) [inline data/building-hours/9-piper-center.yaml]: The Travis failure: 
```json
building-hours/9-piper-center.yaml is invalid
{
  "keyword": "additionalProperties",
  "dataPath": ".schedule[0].hours[1]",
  "schemaPath": "#/additionalProperties",
  "params": {
    "additionalProperty": "to '9:00pm'"
  },
  "message": "should NOT have additional properties"
}
```

Looking at the file, I think you missed the colon between "to" and the time.

File: docs/building-hours.json [hunk 4]
```diff
@@ -2063,6 +2063,399 @@
...
+              "from": "8:00am",
+              "to": "5:00pm"
+            },
+            {
+              "days": [
+                "Tu",
+                "We"
+              ],
+              "from": "7:00pm",
+              "to '9:00pm'": null
```
Comment (hawkrives) [inline docs/building-hours.json]: Yeah. This is what it got translated to.

**Review by hawkrives — CHANGES_REQUESTED** [review]

**Comment by hawkrives** [comment]
Finally, do we want to close #544 before we get:

- Gender & Sexuality Center
- Telecom
- Bon App office

added?

File: data/building-hours/5-2-asc.yaml (hunk 1 above)
Comment (zimmer18) [reply data/building-hours/5-2-asc.yaml:2]: This was flipped to match all the other abbreviations. All the other ones use the way I changed it too.

**Comment by zimmer18** [comment]
Let's close this one and mark it as a completed 2.1 release blocker. I have made a new issue #612 with these remaining three and it won't be a 2.1 release blocker.

File: data/building-hours/5-2-asc.yaml (hunk 1 above)
Comment (hawkrives) [reply data/building-hours/5-2-asc.yaml:2]: Oh, the other ASC places? Good point.

File: data/building-hours/9-osa.yaml (hunk 2 above)
Comment (drewvolz) [reply data/building-hours/9-osa.yaml:2]: I feel that @hawkrives has a decent point. I might flip the name and subtitle. I bet it is thought of as the "SGA office", so an expansion couldn't hurt.

Comment (hawkrives) [reply data/building-hours/9-osa.yaml:2]: Actually, I withdraw my complaint. 

Are there any cases left that use the abbreviation field?

Comment (hawkrives) [reply data/building-hours/9-osa.yaml:2]: The only one left is HOM. 

Could you switch it to Name+subtitle as well, please?

**Review by hawkrives — APPROVED** [review]
LGTM! Thanks!

**Comment by drewvolz** [comment]
Thank you @zimmer18

**Pull request merged**